index.html

```diff
@@ -382,7 +376,7 @@ <h3 id="string_specific_direction">String-specific directional information</h3>
 <p class="advisement" id="bp-using_rlm_lrm">If relying on first-strong heuristics, encourage content developers to use RLM/LRM at the beginning of a string where it is necessary to force a particular base direction, but do not prepend one of these characters to existing strings.</p>
 <p class="advisement" id="bp-rlm_lrm_availability">Do not rely on the availability of RLM/LRM formatting characters in most cases.</p>
 <p>If string data is being provided by users or content developers in web forms or other simple environments, users may not be able to enter these formatting characters.  In fact, most users will probably be unaware that such characters exist, or how to use them.  A web form can render their use unnecessary for immediate inspection if it sets the base direction for the input (which it should).</p>
-<p class="advisement" id="bp-inferring_from_language">If metadata is not available and cannot otherwise be provided, specifications MAY allow a base direction to be <a href="#script_subtag">interpolated from available language metadata</a>.</p>
+<p class="advisement" id="bp-inferring_from_language">(Only) if metadata is not available and cannot otherwise be provided, specifications MAY allow a base direction to be <a href="#script_subtag">interpolated from available language metadata</a>.</p>
```

Comment:
I think this ought to be reversed:

> Specifications SHOULD NOT allow a base direction to be interpolated from available language metadata unless direction metadata is not available and cannot otherwise be provided.

Although the original `MAY` might be closer to being nice about it:

> Specifications that cannot otherwise provide direction metadata or for situations where metadata is not provided MAY allow a base direction to be interpolated from available language metadata.

What do you think?

Reply:
I liked your first alternative, since we need to make it clear that this is not fundamentally a bad approach, rather than an opportunity.